### PR TITLE
Issue 274

### LINKED ISSUE #274: Upgrade Xray with Helm from 2.6.0 to 2.7.3 fails
Upgrading Xray with helm fails with authentication failures against the databases.

Steps to reproduce:
-install Xray: helm install --set common.xrayVersion=2.6.0 xray jfrog/xray
-upgrade Xray: helm upgrade --set common.xrayVersion=2.7.4 --set xray jfrog/xray
-see that the xray-server fails to connect to mongo with invalid credentials

To resolve this issue, you have to exec into each container, grab the password for Mongo/Postgres with printenv, then set it with --set. Note that there is also an issue with rabbitmq connecting, so you have to exec into the rabbitmq container, and set the password to whatever you've set with --set (rabbitmqctl change_password guest ).

helm upgrade --set common.xrayVersion=2.7.4 --set mongodb.mongodbPassword=uyjQvTA0n5 --set postgresql.postgresPassword=uTHbrU3GzP --set rabbitmq.rabbitmqPassword=fDGRicPDOHGd3ZUSoXPUwwfW xray3 jfrog/xray

Notes:
-upgrading with --reuse-values does not help
-installing xray initially with all the passwords set via --set, and then upgrading with all the same passwords set with --set will allow the upgrade to proceed for mongo/postgres, but fail at rabbitmq.

### PR DESCRIPTION
**Pull request by danielezer**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Add information about how to upgrade installations with auto-generated password.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
fixes #274

### CODE REVIEW

File: stable/distribution/README.md [hunk 1]
```diff
@@ -44,6 +44,16 @@ Once you have a new chart version, you can update your deployment with
 helm upgrade distribution jfrog/distribution
 ```
 
+If distribution was installed without providing a value to redis.password (a password was autogenerated), follow these instructions:
+1. Get the current password by running:
+```bash
+REDIS_PASSWORD=$(kubectl get secret -n <namespace> <myrelease>-redis-secret -o jsonpath="{.data.password}" | base64 --decode)
```
Comment (eldada) [inline stable/distribution/README.md]: Need also to extract postgresql password

File: stable/mission-control/README.md [hunk 2]
```diff
@@ -105,6 +105,16 @@ Once you have a new chart version, you can update your deployment with
 helm upgrade mission-control jfrog/mission-control
 ```
 
+If mission-control was installed without providing a value to postgresql.postgresPassword (a password was autogenerated), follow these instructions:
```
Comment (eldada) [inline stable/mission-control/README.md]: JFMC already has a *Auto generated passwords* section that covers this.

File: stable/xray/README.md [hunk 3]
```diff
@@ -47,6 +47,16 @@ To upgrade an existing Xray, you still use **helm**
 helm upgrade --set common.xrayVersion=2.1.2 jfrog/xray
 ```
 
+If xray was installed without providing a value to postgresql.postgresPassword (a password was autogenerated), follow these instructions:
+1. Get the current password by running:
+```bash
+POSTGRES_PASSWORD=$(kubectl get secret -n <namespace> <myrelease>-postgresql -o jsonpath="{.data.postgres-password}" | base64 --decode)
```
Comment (eldada) [inline stable/xray/README.md]: Need to also extract rabbitMQ and mongoDB passwords

Comment (danielezer) [reply stable/xray/README.md]: Done

File: stable/mission-control/README.md (hunk 2 above)
Comment (danielezer) [reply stable/mission-control/README.md]: Removed

File: stable/distribution/README.md (hunk 1 above)
Comment (danielezer) [reply stable/distribution/README.md]: Done

**Pull request merged**